src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,27 @@
+package codeu.chat.common;
+
+import java.io.IOException;
+
+import codeu.chat.util.Uuid;
+public final class ServerInfo {
+  private final static String SERVER_VERSION = "1.0.0";
+
+  public final Uuid version;
+  public ServerInfo(){
+      Uuid temp = null;
+      try {
+        temp = Uuid.parse(SERVER_VERSION);
```

Comment:
SERVER_VERSION is final and static from the time it's initialized in the class, so it's a constant value that will never change. I should also point out that any time a constant like this is initialized, they usually have NAMES_LIKE_THIS . Because of these two things, a try/catch/finally is unnecessary in the constructor because if a value never changes to something it shouldn't be and is always initialized when it needs to be, it will be in a constant, non-erroneous state 24/7.